uml/uml_description.txt

```diff
@@ -0,0 +1,13 @@
+There are 2 players in each game. The game is turn-based, each turn player can take some actions: play cards from his hand (many times a turn), use hero power (once a turn), attack with his minions (once a turn each), attack with his hero (once a turn).
```

Comment:
Cool idea to wrap all of this up.
Need to think how we want to approach triggered events and actions – looks like it's gonna be pain to handle all of this